templates/web/base/report/form/private_details.html

```diff
@@ -0,0 +1,18 @@
+<!-- private_details.html -->
+<h2 class="form-section-heading form-section-heading--private">[% loc('Tell us about you') %]</h2>
+
+[% UNLESS c.user_exists %]
+<p class="hidden-nojs js-new-report-sign-in-hidden">
+    <a id="js-new-report-show-sign-in" href="#">[% loc('Or sign in with password to prefill this information.') %]</a>
```

Comment:
This was moved here in 15c249abdec934d416a2ba49a66271b0bd0002b7 but this means it no longer is available to the update flow at all, there's no way of logging in with password there any more.